docs/CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## Unreleased
 ### Added
 - Add new flaw reference type "UPSTREAM"
+- Auto-reset CVSS validation flag on NVD CVSS removal (OSIDB-3407)
```

Comment:
I am a bit hesitant here as there was no real bug. Two so far independent processes could lead into a mutually invalid state which we do not think about before but none of them is broken on its own. If you have a stronger opinion, I can move it but otherwise to me it seems that I am adding an extra automatic step to keep the data consistency but I am not fully convinced that the collector was buggy before.

Reply:
No problem, my comment was based on a Jira issue, which says bug :)